src/TulsiGenerator/Scripts/bazel_build.py

```diff
@@ -1556,6 +1563,17 @@ def _ExtractTargetSourcePaths(self):
 
     return source_path_prefixes
 
+  def _LinkTulsiWorkspace(self):
+    """Links the Bazel Workspace to the Tulsi Workspace ( `tulsi-workspace` )"""
+    bazel_workspace = self.bazel_symlink_prefix + os.path.basename(self.workspace_root)
+    tulsi_workspace = self.workspace_root + "/tulsi-workspace"
+    if os.path.islink(tulsi_workspace):
+      os.unlink(tulsi_workspace)
+    os.symlink(bazel_workspace, tulsi_workspace)
+    if not os.path.exists(tulsi_workspace):
```

Comment:
Originally, the error was printed at the call site of `_LinkTulsiWorkspace` - I can move it here and improve it a bit.